dev/start.bat

```diff
@@ -72,13 +72,21 @@
 @ECHO.
 @ECHO Note: This choice can be overridden by setting the 'installer_type' key in ~\.condarc.
 @ECHO.
+@CALL :RESET_ERRORLEVEL
 @SET /P "_INSTALLER_TYPE=Enter choice [1]: "
-@IF NOT %ErrorLevel%==0 @EXIT /B 1
+@SET "_INSTALLER_TYPE_PROMPT=%ErrorLevel%"
+@SET "_INSTALLER_TYPE_RAW=%_INSTALLER_TYPE%"
 :: normalize user input
 @IF "%_INSTALLER_TYPE%"=="1" @SET "_INSTALLER_TYPE=miniconda"
 @IF "%_INSTALLER_TYPE%"=="" @SET "_INSTALLER_TYPE=miniconda"
```

Comment:
Since there is no way to distinguish between an `Enter` and `Ctrl-C` (both result in `%_INSTALLER_TYPE%==""` **and** `%ErrorLevel%==1`) we need to just pick one behavior:

```suggestion
:: terminated or empty prompt returns errorlevel 1, set default
@IF %ErrorLevel%==1 @SET "_INSTALLER_TYPE=miniconda"
@CALL :RESET_ERRORLEVEL
:: normalize user input
@IF "%_INSTALLER_TYPE%"=="1" @SET "_INSTALLER_TYPE=miniconda"
```